scout/server/blueprints/cases/templates/cases/case.html

```diff
@@ -49,9 +49,45 @@
         <div class="row">
             <div class="col-sm-4">
               <h4 class="">
-                  Case: {{case.display_name}}<br>
-                  Status: {{case.status}}<br>
-                  <span class="badge bg-sm bg-secondary">Internal ID: {{case._id}}</span>
+                Case: {{case.display_name}} <span class="badge bg-sm bg-secondary">{{case._id}}</span><br>
+                Status: {{case.status}}
+                <form method="POST" action="{{ url_for('cases.status', institute_id=institute._id, case_name=case.display_name) }}">
```

Comment:
A little crowded and slow to read here. Perhaps paint the `case.status` according to status colors, and have a just little more space between the form and the current status?

Reply:
Agreed, will fix!